src/dymaptic.GeoBlazor.Core/Components/Renderers/ColorRamps/ColorRamp.cs

```diff
@@ -0,0 +1,50 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps;
+using dymaptic.GeoBlazor.Core.Serialization;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+namespace dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps;
+
+/// <summary>
+///     A ColorRamp object is used to specify a range of colors that are applied to a group of symbols or pixels.
+///     There are two types of color ramps available:
+///     Algorithmic color ramp: A AlgorithmicColorRamp is defined by two colors and the algorithm used to traverse the intervening color space between them.
+///     Multipart color ramp: A MultipartColorRamp is defined by a list of constituent color ramps.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-ColorRamp.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public abstract class ColorRamp : MapComponent
+{
+    public ColorRamp() { }
+
+    public ColorRamp(ColorRampType colorRampType, MultipartColorRamp colorRamps)
```

Comment:
abstract classes don't need constructors at all, unless you're going to call `: base()` from the inheriting class and pass values.

Reply:
corrected and removed.  noted for future classes